cekit/cli.py

```diff
@@ -462,6 +463,13 @@ def run(self, clazz):
             sys.exit(0)
         except KeyboardInterrupt:
             pass
+        except CalledProcessError as ex:
+            LOGGER.error(
```

Comment:
The first `{}` is specially handled by `CalledProcessError`. An example:
```
ERROR Command 'git' returned non-zero exit status 1. Command stdout is 'A GIT ERROR' with stderr 'None'
```

Reply:
Right, what I'm looking after is basically having a separate, probably 3 calls in this case, to the `LOGGER.error`.

What you posted above is fine, for simple cases. I imagine that there will be cases where a pretty long, multi-line error will be passed and the formatting of it this way could lead to problems understanding what is the original one vs. what was added by us vs. what is still part of stdout and what is stderr now.

Reply:
Potentially yes. At first I was thinking that why to log empty stuff, but I guess in case of failures, making it explicit that the output was empty is a good thing.

I imagine that it could be like: `"Standard output:\n{}"`. Then we know that the content below the line is original. Similar approach to error.